.github/workflows/backend-cd-dev.yml

```
@@ -4,13 +4,19 @@ on:
     paths: [ 'backend/**' ]
+  pull_request:
+    branches: [ backend ]
   workflow_dispatch:
 
 jobs:
   build-and-deploy:
     runs-on: [self-hosted, dev-runner]
 
     steps:
+      - name: Initialize workspace permissions
+        run:|
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 17: could not find expected ':'

(syntax-check)